etc/behat/services.xml

```diff
@@ -30,6 +30,7 @@
         <parameter key="sylius.behat.page.admin.crud.create.class">Sylius\Behat\Page\Admin\Crud\CreatePage</parameter>
         <parameter key="sylius.behat.page.admin.crud.update.class">Sylius\Behat\Page\Admin\Crud\UpdatePage</parameter>
         <parameter key="sylius.behat.service.accessor.notification_accessor.class">Sylius\Behat\Service\Accessor\NotificationAccessor</parameter>
+        <parameter key="sylius.behat.service.notification_validator.class">Sylius\Behat\Service\NotificationValidator</parameter>
```

Comment:
IMO `sylius.behat.service.notification_{validator,accessor}` seems the best for me, as grouping them by accessor / validator doesn't really help - I need a notification validator, not a validator for notifications.